.gitmodules

```diff
@@ -3,4 +3,13 @@
 	url = https://github.com/foundry-rs/forge-std
 [submodule "template/contracts/lib/openzeppelin-contracts"]
 	path = template/contracts/lib/openzeppelin-contracts
-	url = https://github.com/Openzeppelin/openzeppelin-contracts
+	url = https://github.com/OpenZeppelin/openzeppelin-contracts
+[submodule "template/contracts/lib/ERC721A"]
```

Comment:
These would need to be added here as well: https://github.com/coinbase/build-onchain-apps/blob/main/src/utils/git.ts#L19 so that `.gitmodules` is properly setup for anyone installing the package using `npx @coinbase/build-onchain-apps@latest create`

Reply:
aye aye!

Reply:
Awesome! Thanks